cogs/subscribers.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) idoneam (2016-2019)
+#
+# This file is part of Canary
+#
+# Canary is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Canary is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Canary. If not, see <https://www.gnu.org/licenses/>.
+
+# discord-py requirements
+import discord
+from discord.ext import commands
+from discord import utils
+import asyncio
+
+# URL access and parsing
+from bs4 import BeautifulSoup
+
+# Other utilities
+import os
+import re
+import pickle
+import feedparser
+import requests
+
+CFIA_FEED_URL = "http://inspection.gc.ca/eng/1388422350443/1388422374046.xml"
+METRO_STATUS_API = "https://www.stm.info/en/ajax/etats-du-service"
+
+# Default values by line number for status
+# Integers in list are line colours
+METRO_STATUS = {
```

Comment:
since this gets changed (i.e. not constant), don't use all uppercase https://www.python.org/dev/peps/pep-0008/#constants

Reply:
will do!

Reply:
wrong change